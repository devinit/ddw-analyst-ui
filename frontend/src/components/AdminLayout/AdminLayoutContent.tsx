import * as React from 'react';

export const AdminLayoutContent: React.SFC = props => {
  return (
    <React.Fragment>
      { props.children }
    </React.Fragment>
  );
};
