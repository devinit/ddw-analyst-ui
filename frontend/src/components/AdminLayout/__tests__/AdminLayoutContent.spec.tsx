import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { AdminLayoutContent } from '../AdminLayoutContent';

test('renders the specified children', () => {
  const renderer = TestRenderer
    .create(<AdminLayoutContent>My Content</AdminLayoutContent>)
    .toJSON();

  expect(renderer).toMatchSnapshot();
});

test('renders no content when no children are added', () => {
  const renderer = TestRenderer
    .create(<AdminLayoutContent/>)
    .toJSON();

  expect(renderer).toMatchSnapshot();
});
