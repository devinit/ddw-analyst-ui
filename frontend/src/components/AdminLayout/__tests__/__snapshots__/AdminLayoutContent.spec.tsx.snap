// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders no content when no children are added 1`] = `null`;

exports[`renders the specified children 1`] = `"My Content"`;
